### PR TITLE
Local Events Feed (Happenings) - Revision

### PR DESCRIPTION
**Pull request by zsyed1**

### CODE REVIEW

**Comment by zsyed1** [comment]
@jmmastey I made the Ruby changes you suggested to the application views, but did not spend much time on fixing the CSS. I plan to use pmail for that purpose. Ashton has given me similar comments on my CSS there.

**Comment by djkotowski** [comment]
I recognize many of these wallpapers.

File: local_events_feed/app/helpers/application_helper.rb [hunk 1]
```diff
@@ -0,0 +1,51 @@
+module ApplicationHelper
+
+  include ShowingsHelper
```
Comment (jmmastey) [inline local_events_feed/app/helpers/application_helper.rb]: It's a little weird for ApplicationHelper to include another Helper module (unless it's another truly global helper, I suppose). Not clear here.

File: local_events_feed/app/helpers/application_helper.rb [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+    calendar.add_event(showing.to_ics)
+    calendar.publish
+    calendar.to_ical
+  end
+
+  def get_local_time_view(time)
+    LocalTimeView.new(time)
+  end
+
+  class LocalTimeView
```
Comment (jmmastey) [inline local_events_feed/app/helpers/application_helper.rb]: I actually like this internal class here.

File: local_events_feed/app/helpers/showings_helper.rb [hunk 3]
```diff
@@ -0,0 +1,53 @@
+require 'theatre_in_chicago/event'
```
Comment (jmmastey) [inline local_events_feed/app/helpers/showings_helper.rb]: This shouldn't be necessary.

File: local_events_feed/app/views/application/_user.html.erb [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+	  <td>
+		<div class="add-to-cal">
+		  <%= link_to image_tag("calendar.png", :class => 'add-image tooltip', :title => 'Add Calendar'),
+		  showing_path(showing, format: :ical), :id => "add_cal-#{showing.id}" %>
+		</div>
+	  </td>
+	  <td class="attributes">
+		<ul>
+		  <li>
+			<% local_time_view = ApplicationHelper::LocalTimeView.new(showing.time) %>
```
Comment (jmmastey) [inline local_events_feed/app/views/application/_user.html.erb]: I keep thinking you may be able to access this without the ApplicationHelper namespace. I believe the ApplicationHelper should be included in this scope, which would shorten these a bit.

**Comment by jmmastey** [comment]
Hey, sorry this took so long. Added a few notes, but this looks good! Thanks.

**Pull request closed without merging**